actioncable/app/assets/javascripts/action_cable/consumer.coffee

```diff
@@ -22,6 +33,12 @@ class ActionCable.Consumer
   send: (data) ->
     @connection.send(data)
 
+  start: ->
+    @connection.open()
+
+  stop: ->
+    @connection.close(true)
```

Comment:
Passing `true` reads really weird. I think adding a new API could capture this better, perhaps `close(withoutReconnect: true)` could do (if I'm understanding the purpose of the argument right).